src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -59,6 +59,8 @@ private readonly Dictionary<RestoreTargetGraph, Dictionary<string, LibraryInclud
         private const string TargetFrameworksCount = nameof(TargetFrameworksCount);
         private const string RuntimeIdentifiersCount = nameof(RuntimeIdentifiersCount);
         private const string TreatWarningsAsErrors = nameof(TreatWarningsAsErrors);
+        private const string SDKAnalysisLevel = nameof(SDKAnalysisLevel);
+        private const string UsingMicrosoftNETSdk = nameof(UsingMicrosoftNETSdk);
```

Comment:
We should also include `NETCoreSdkVersion`, otherwise we have no way guess if the customer is using the default `SDKAnalysisLevel` or if they explicitly set a different value.  Without being able to estimate explicit SDKAnalysisLevel usage, I don't know what the point of collecting the telemetry is.